pages/services/beta-dcos-monitoring/0.4.3-beta/operations/prometheus/storage/index.md

```diff
@@ -26,3 +26,27 @@ You can configure the storage retention (default: 15d) of Prometheus by setting
 
 The above configuration will tell Prometheus to remove old data after 30 days.
 Refer to this [document](https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects) for more details about storage retention.
+
+## DSS Volume
+
+You can configure Prometheus to use a dedicated volume provided by the [DC/OS Storage Service](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/) (DSS).
+
+At least one volume must be available for Prometheus to use. For more on creating volumes, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-volume/dcos-storage-volume-create).
+
+When configuring Prometheus, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume. For more on volume profiles, see the [DSS documentation](https://docs.mesosphere.com/services/beta-storage/0.5.3-beta/cli-references/dcos-storage-profile/).
+
+The following configuration will deploy Prometheus with a `fast` DSS volume that is 25 GB or larger. If there is no such volume available, Prometheus will fail to deploy.
```

Comment:
Is there any minimal requirements for this volume that we should recommend? Should Prometheus volumes be on high performance storage (e.g. SSD backed)? Is the example size below (512MB) sufficient? Or should it be larger. I would definitely include a recommendation for storage guidelines. I would also align the example to your guidelines.